porick/views.py

```diff
@@ -148,7 +150,7 @@ def signup():
         create_user(username, password, email)
         authenticate(username, password)
         g.user = User.query.filter(User.username == username).first()
-        return render_template('/signup_success.mako')
+        return render_template('/signup_success.html')
```

Comment:
wtf, how did this work when I created my account on my local install for testing? weird